src/app.tsx

```diff
@@ -1,6 +1,6 @@
 // lib imports
-import * as React from 'react';
-import * as ReactDOM from 'react-dom';
+import React from 'react';
```

Comment:
Hrmm. I had a lot of problems with this. I updated https://github.com/systemjs/builder/issues/775 to see if i can get some help to get the namespace import working again.